### PR TITLE
Increase KafkaIO version to 0.2.0

### PR DESCRIPTION
**Pull request by rangadi**
Recent PR #491 changes how KafkaIO splits. This makes it incompatible
with Dataflow update across these two versions.

### CODE REVIEW

**Comment by rangadi** [comment]
+R: @dhalperi 
This bumps version number. I was sure where I should include the release notes. We don't yet have a README for kafka directory.

**Comment by dhalperi** [comment]
Suggest creating a README.md

**Comment by rangadi** [comment]
Added README.md.
+R: dhalperi

File: contrib/kafka/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+KafkaIO provides unbounded sources and sinks for [Kafka](https://www.firebase.com/)
+topics. Kafka version 0.9 and above are supported.
+
+## Basic Usage
+
+* Read from a topic with 8 byte long keys and string values:
+```java
+ PCollection<KV<Long, String>> kafkaRecords =
+   pipeline
+     .applY(KafkaIO.read()
```
Comment (dhalperi) [inline contrib/kafka/README.md]: lowercase `y`

File: contrib/kafka/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+     );
+```
+
+* Write the same PCollection to a Kafka topic:
+```java
+ kafkaRecords.apply(KafkaIO.write()
+   .withBootstrapServers("broker_1:9092,broker_2:9092")
+   .withTopic("results")
+   .withKeyCoder(BigEndianLongCoder.of())
+   .withValueCoder(StringUtf8Coder.of())
```
Comment (dhalperi) [inline contrib/kafka/README.md]: missing last line?

File: contrib/kafka/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# KafkaIO : Dataflow Unbounded Source and Sink for Kafka Topics
+
+KafkaIO provides unbounded sources and sinks for [Kafka](https://www.firebase.com/)
+topics. Kafka version 0.9 and above are supported.
```
Comment (dhalperi) [inline contrib/kafka/README.md]: is this correct, or are code changes needed to support Kafka 0.10 or 0.11?

File: contrib/kafka/README.md (hunk 1 above)
Comment (rangadi) [inline contrib/kafka/README.md]: Done

File: contrib/kafka/README.md (hunk 3 above)
Comment (rangadi) [inline contrib/kafka/README.md]: Works with 0.10, but haven't tested with 0.11 (kafka master?). We can remove the line or mention specific versions tested.

File: contrib/kafka/README.md (hunk 2 above)
Comment (rangadi) [inline contrib/kafka/README.md]: Done.

**Comment by rangadi** [comment]
+R: @dhalperi . I had addressed the comments immediately. I am not sure how to mark 'Changes Requested' as 'Done'.

**Pull request merged**